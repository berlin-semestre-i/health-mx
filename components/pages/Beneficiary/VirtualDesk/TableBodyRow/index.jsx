import React from 'react'
import { Table } from 'semantic-ui-react'
import Link from 'next/link'

const TableBodyRow = ({ dependant }) => (
  <Table.Row>
    <Table.Cell>{dependant.relationship}</Table.Cell>
    <Table.Cell><Link href="#"><a>{dependant.name}</a></Link></Table.Cell>
    <Table.Cell>{dependant.curp}</Table.Cell>
    <Table.Cell>{dependant.sex}</Table.Cell>
    <Table.Cell>{dependant.age}</Table.Cell>
    <Table.Cell>{dependant.situation}</Table.Cell>
  </Table.Row>
)

export default TableBodyRow
