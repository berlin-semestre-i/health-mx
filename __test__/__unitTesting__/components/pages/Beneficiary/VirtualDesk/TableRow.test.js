import React from 'react'
import { shallow, mount } from 'enzyme'
// eslint-disable-next-line max-len
import TableBodyRow from '../../../../../../components/pages/Beneficiary/VirtualDesk/TableBodyRow/index'

const testDependant = {
  relationship: 'CÃ“NYUGUE',
  name: 'Maria Antonieta',
  curp: 'JHSA21092109201',
  sex: 'M',
  age: '23',
  situation: 'Vigente',
}

describe('Table Row', () => {
  it('renders', () => {
    const component = shallow(<TableBodyRow dependant={testDependant} />)
    expect(component).toMatchSnapshot()
  })
})