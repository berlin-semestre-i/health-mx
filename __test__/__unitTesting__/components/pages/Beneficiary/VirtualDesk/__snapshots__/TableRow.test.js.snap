// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table Row renders 1`] = `
<TableRow
  as="tr"
  cellAs="td"
>
  <TableCell
    as="td"
  >
    CÃ“NYUGUE
  </TableCell>
  <TableCell
    as="td"
  >
    <Link
      href="#"
    >
      <a>
        Maria Antonieta
      </a>
    </Link>
  </TableCell>
  <TableCell
    as="td"
  >
    JHSA21092109201
  </TableCell>
  <TableCell
    as="td"
  >
    M
  </TableCell>
  <TableCell
    as="td"
  >
    23
  </TableCell>
  <TableCell
    as="td"
  >
    Vigente
  </TableCell>
</TableRow>
`;
